internal/pkg/stream/cloudformation.go

```diff
@@ -77,9 +80,16 @@ func (s *StackStreamer) Fetch() (next time.Time, err error) {
 			StackName: aws.String(s.stackName),
 		})
 		if err != nil {
+			// Check for throttles and wait to try again using the StackStreamer's interval.
+			s.retries += 1
```

Comment:
nit: should we move this statement to inside the if-statement so that it's only incremented on retryable errors.